enterprise/backend/test/metabase_enterprise/sandbox/models/params/field_values_test.clj

```diff
@@ -45,7 +47,23 @@
             (params.field-values/get-or-create-advanced-field-values!
              :sandbox
              (db/select-one Field :id (mt/id :categories :id)))
-            (is (= 1 (db/count FieldValues :field_id categories-id :type fv-type)))))))
+            (is (= 1 (db/count FieldValues :field_id categories-id :type fv-type))))
+
+          (testing "after changing the question, should create new FieldValues"
+            (let [new-query (mt/mbql-query categories
+                                           {:filter [:and [:> $id 1] [:< $id 4]]})]
+              ;; sleeping should ensure that updated_at changes
+              (Thread/sleep 1000)
```

Comment:
MySQL seems to have second granularity, which makes this test somewhat slow. This also means that if the sandboxed user uses the sandboxed query within a second after the modification of the query, they will. get wrong filter values. I don't think this is a problem in practice.

Reply:
Yeah, `timestamp` in MySQL is second granularity by default. You should just add a migration to change that column to `timestamp(3)` (millisecond) or `timestamp(6)` (nanosecond) so we don't have to worry about this in the future. We've done it a bunch of times in the past to fix the same situation with other columns, so there's prior art in the migration file that you could refer to

Reply:
I suppose `timestamp_type` (microseconds) is the right type to set. I'll update the PR.